File: hw/santorum/u3/hw32Strategy/Strategy/src/main/java/ec/edu/espe/strategy/model/BubbleSort.java
package ec.edu.espe.strategy.model;

import java.util.Arrays;

/**
 *
 * @author Thais Santórum
 */
public class BubbleSort implements SortingStrategy {
    
    public int [] sort (int data[]){
        System.out.println("Selected strategy BubbleSort");
        int[] arr = data.clone();
        Arrays.sort(arr);
        return arr;   
    }
}

File: exam/vargas/u3/q47_100/javaScript/model/BubbleSort.js
const SortingStrategy = require("./SortingStrategy");

class BubbleSort extends SortingStrategy {
  sort(data) {
    console.log("Selected strategy BubbleSort");
    return [...data].sort((a, b) => a - b);
  }
}

module.exports = BubbleSort;
